.travis.yml

```diff
@@ -7,7 +7,7 @@ sudo: false
 
 cache:
   directories:
-    - $HOME/.composer/cache
+    - $HOME/.composer/cache/files
```

Comment:
Yes, it is better cache this folder

https://github.com/FriendsOfSymfony/FOSRestBundle/blob/2.0/.travis.yml#L13
https://github.com/liip/LiipImagineBundle/blob/master/.travis.yml